book/src/pong-tutorial/contribution.md

```diff
@@ -1,9 +1,28 @@
-# Call for contribution
+# Tutorial completed!
 
 And... that's where the pong chapter ends. We hope you found it useful!
 
 You can find the entire code with balls, score and music on the example pages available [here](https://github.com/amethyst/amethyst/tree/master/examples/pong).
 
-For more examples from the community you can check out this list of [Games made with Amethyst](https://community.amethyst.rs/t/games-made-with-amethyst/134). You can get additional help by leaving a post on [our forum](https://community.amethyst.rs) or on our [Discord server](https://discord.gg/amethyst).
+Next up in this book we will explain how Amethyst does Math, Input, Assets and so on. Whenever you have a need for more learning-by-example materials just come back to this page for an overview of available resources.
+
+## Other tutorials or examples
+
+[Amethyst Quickstarter](https://github.com/amethyst/amethyst-starter-2d)
```

Comment:
Likely should either add two trailing spaces or an additional newline
(markdown [collapses it](https://github.com/amethyst/amethyst/blob/11f693dae103ea7731d071cef4759f1cf2757933/book/src/pong-tutorial/contribution.md#other-tutorials-or-examples) into one line without either of those)